environment.yml

```diff
@@ -8,7 +8,7 @@ dependencies:
   - pip
   - pip:
     - NDBC
-    - git+https://github.com/Deltares/hydromt_fiat.git@d1b6473
+    - git+https://github.com/Deltares/hydromt_fiat.git@fiat_integrator
```

Comment:
shouldn't this be changed in the pyproject.toml file as well?

Reply:
yeah that is correct